src/main/java/io/cdap/plugin/kudu/KuduSinkConfig.java

```diff
@@ -260,4 +317,139 @@ public int getSeed() {
   public int getThreads() {
     return (optBossThreads != null) ? Integer.parseInt(optBossThreads) : 1;
   }
+
+  public void validate(FailureCollector failureCollector) {
+    if (Strings.isNullOrEmpty(optSchema)) {
```

Comment:
Control this with @Nullable. No checks for empty values should be present. Nullable annotation does it for us. As far as I understand it.

Reply:
Done